Project2/style.css

```diff
@@ -0,0 +1,25 @@
+.card {
+    max-width: 500px;
+    margin: 10px auto;
+    background-color: #ffffff;}
+figure {margin: 0 auto;}
+        max-width: 500px;
```

Comment:
This `max-width` declaration is **not** inside a ruleset and can create broken CSS. Is it for the `figure`? If so, what you want is:

```css
figure {
    margin: 0 auto;
    max-width: 500px;
}
```
If not,  you should either move it to the ruleset it belongs to or delete it.